cmd/commands/common.go

```diff
@@ -73,8 +74,12 @@ var (
 		return r, repofs, nil
 	}
 
-	glob = func(fs fs.FS, pattern string) ([]string, error) {
-		return billyUtils.Glob(fs, pattern)
+	removeAll = billyUtils.RemoveAll
+
+	glob = billyUtils.Glob
+
+	writeFile = func(fs billy.Basic, filename string, data []byte) error {
```

Comment:
I think that this way of assigning a function to a var to later replace in tests should be a last resort. It's really hard to manage and we could end up with a file with tons of unrelated functions assigned to vars.
For example - `writeFile` `glob` `removeAll` should all be just tested using a `memfs`. It's easier and cleaner to do than to replace the functions before and after every test.

Reply:
Looking at the current implementation of `billyUtils.Glob` and `billyUtils.RemoveAll`, they seem like they will not be easily mocked by a fsmock.
I agree that WriteFile is much simpler, and we even had our own implementation of it before we noticed billyUtils, so we can easily revert to it.

Reply:
My point is - we don't want to mock `Glob` and `RemoveAll`, let them do their job, just prepare a `memfs` for your tests with the files you need and later assert on the memfs for the expected changes

Reply:
I did that on a couple of tests on `repo_test.go` you can take a look

Reply:
ok, I'll take a look on how to do that